itest/itest.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash -eE
+
+export FIFO_NAME=itest/itest_output_$$
```

Comment:
Maybe this should be added to the `.gitignore` to avoid accidental commits in case the cleanup fails

Reply:
Good call, done

Reply:
Good call, done